src/store/workflows.module.js

```diff
@@ -40,6 +40,28 @@ function _getCycles (workflows) {
   return cycles
 }
 
+/**
+ * Compute percent progress.
+ *
+ * @see https://github.com/cylc/cylc-flow/blob/de7d938496e82dbdfb165938145670dd8e801efd/lib/cylc/gui/updater_tree.py#L248-L263
+ * @param {number} startedTime in milliseconds since 1970-01-01 00:00:00 UTC, e.g. 1568353099874
+ * @param {number} meanElapsedTime mean elapsed time (e.g. 13.5)
+ * @returns {number} the percent progress, e.g. 25 (meaning 25% progress)
+ * @private
+ */
+function _computePercentProgress (startedTime, meanElapsedTime) {
+  const now = Date.now()
+  // startedTime > now reportably possible via interaction with `cylc reset`
+  if (startedTime > now || meanElapsedTime === 0) {
+    return 0
+  }
+  const elapsedTime = startedTime + meanElapsedTime
```

Comment:
Found the problem: `startedTime` is **milliseconds** since 1970; meanElapsedTime is **seconds**!  Fixed in my environment, then:
![shot](https://user-images.githubusercontent.com/2362137/64929559-c6d5fb00-d87b-11e9-92a6-7aa3a02b51aa.png)

Reply:
Thanks a ton @hjoliver ! Feel free to push to this branch if you'd like. Then I can sync and test and if it works and tests pass, move it to ready-to-review 🎉

Reply:
Will do...

Reply:
Done